samples/treectrl/treetest.cpp

```diff
@@ -1055,7 +1055,7 @@ void MyTreeCtrl::CreateStateImageList(bool del)
     AssignStateImageList(states);
 }
 
-#if USE_GENERIC_TREECTRL || !defined(__WXMSW__)
+#if USE_GENERIC_TREECTRL || (!defined(__WXMSW__) && !defined(__WXQT__))
```

Comment:
It seems wrong to change it here but not in the other lines testing `USE_GENERIC_TREECTRL`. I'm not sure about the logic here, to be honest, but I think we really need to define `USE_MSW_TREECTRL` once and then just test for it as the intention seems to have some MSW-specific code only.

To test whether this works correctly, you need to change `USE_GENERIC_TREECTRL` to 1 in `samples/treectrl/treetest.h` and build the sample -- does it compile/work?